testing/src/main/java/io/grpc/testing/GrpcCleanupRule.java

```diff
@@ -175,6 +175,7 @@ public void evaluate() throws Throwable {
    * Releases all the registered resources.
    */
   private void teardown() {
+    stopwatch.reset();
```

Comment:
If you reset `stopwatch`, I guess you should also reset all other fields such as `firstException` and `resources`.

Reply:
Looks like `resources` is being cleared by the `teardown` method already, so only `firstException` is being left?

Reply:
`resources` won't be cleared if `teardown()` throws a runtime exception. But I'm okay with the current change.